pkg/utils/flow_utils.go

```diff
@@ -260,6 +258,47 @@ func PacketSize(f *flow.Flow) uint64 {
 	return k.Bytes
 }
 
+func AddDropReason(f *flow.Flow, dropReason uint32) {
+	if f == nil {
+		return
+	}
+
+	f.Verdict = flow.Verdict_DROPPED
+	f.EventType = &flow.CiliumEventType{
+		Type:    int32(api.MessageTypeDrop),
+		SubType: int32(api.TraceToNetwork), // This is a drop event and direction is determined later.
+	}
+	switch dropReason {
+	case 0:
+		f.DropReasonDesc = flow.DropReason_POLICY_DENIED
+	case 1:
+		f.DropReasonDesc = flow.DropReason_NAT64
```

Comment:
nat64 is actually incorrect, nat 64 is a NTA for converting v4 addrs to v6, is there any other NAT related issue ?